src/modules/transaction/utils/encoding.js

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import { codec } from '@liskhq/lisk-codec';
+import { utils } from '@liskhq/lisk-cryptography';
+
+// TODO: Use from service endpoint/elements
```

Comment:
why dont we implement the `useSchemaQuery` hook instead? (`GET /commands/parameters/schemas`)

Reply:
I think once SDK exposes the schema via elements we can use the schema from elements.

Reply:
ok